lib/plugins/common/json-rpc.js

```diff
@@ -2,32 +2,37 @@
 const axios = require('axios')
 const uuid = require('uuid')
 const fs = require('fs')
+const _ = require('lodash/fp')
 
 module.exports = {fetch, parseConf}
 
 function fetch (account, method, params) {
-  const data = {
-    method,
-    params,
-    id: uuid.v4()
-  }
-
-  return axios({
-    method: 'post',
-    auth: {username: account.username, password: account.password},
-    url: `http://localhost:${account.port}`,
-    data
-  })
+  return Promise.resolve(true)
```

Comment:
it was to bundle the first lines in a promise so that if there is any error with the arguments it can be raised inside that.
Otherwise you would need to try{}catch{} the function call to be completely safe

Reply:
Ok, makes sense.